sdks/python/apache_beam/transforms/__init__.py

```diff
@@ -23,3 +23,8 @@
 from apache_beam.transforms.ptransform import *
 from apache_beam.transforms.timeutil import TimeDomain
 from apache_beam.transforms.util import *
+
+try:
```

Comment:
<!--new_thread; commit:dd3ae5364c173a99c8a15f7e80960b9307993f7c; resolved:0-->
Please add comment: no backwards compatibility guarantees here.

Reply:
So basically that is, I'm supposed to add this 'no backwards compatibility guarantees here.' in everywhere I touched, right?

Reply:
No, just here--this is part of the public API; I think the other places you added things already have this comment.